### PR TITLE
DNM - FIPS testing with ansible-test-base

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gosriniv@redhat.com>

### CODE REVIEW

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 - job:
```
Comment (pabelanger) [inline zuul.d/jobs.yaml:2]: okay, we should now add the new add-build-sshkey vars here, to generate ecdsa keys, and see what breaks.

File: playbooks/ansible-network-vyos-appliance/files/bootstrap.yaml [hunk 2]
```diff
@@ -9,7 +9,8 @@
   tasks:
     - name: lookup SSH public key
       set_fact:
-        _ssh_key: "{{ lookup('file', '~/.ssh/id_rsa.pub') }}"
+        _ssh_file: "~/.ssh/{{ zuul_ssh_key_dest }}.pub"
```
Comment (pabelanger) [inline playbooks/ansible-network-vyos-appliance/files/bootstrap.yaml:12]: zuul variables are out of scope here, i think in this case, it is okay to replace rsa with ecdsa hardcoded

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
Closing due to rebase issues. Opened https://github.com/ansible/ansible-zuul-jobs/pull/817

**Pull request closed without merging**